agent/health_endpoint.go

```diff
@@ -203,10 +192,6 @@ func (s *HTTPHandlers) healthServiceNodes(resp http.ResponseWriter, req *http.Re
 		args.TagFilter = true
 	}
 
-	if _, ok := params["merge-central-config"]; ok {
```

Comment:
@kisunji : same question as https://github.com/hashicorp/consul/pull/13256/files#r882931918 - is this delete supposed to be here?